MainTest.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.Test;
+import static org.junit.Assert.*; 
+
+public class MainTest{
+
+  @Test
+  public void testNothing(){}
+
+  @Test
+  public void testSomeStuff(){
+    assertEquals(5, 2+3);
```

Comment:
Bitte ergänzen Sie noch aussgekräftige Meldungen, falls die Zusicherungen schief gehen. Schauen Sie dazu in die Dokumentation von JUnit.